compiler/src/dotty/tools/dotc/parsing/Parsers.scala

```diff
@@ -404,14 +404,13 @@ object Parsers {
 
     var opStack: List[OpInfo] = Nil
 
-    def checkAssoc(offset: Int, op: Name, leftAssoc: Boolean) =
+    def checkAssoc(offset: Int, op: Name, leftAssoc: Boolean, op2: Name) =
```

Comment:
I'm assuming you want to avoid a larger diff, but it looks weird that `op2` is at the end. If you want to refactor that - put it in a second commit and we'll merge both of them unsquished.